renovate.json

```diff
@@ -1,5 +1,6 @@
 {
-  "extends": ["github>canonical/renovate-websites"],
+  "dependencyDashboard": true,
+  "extends": ["github>canonical/renovate-websites", "schedule:monthly"],
```

Comment:
Somewhere along the line we lost our monthly schedule (I was wondering why we weren't getting renovate PRs). I'm not sure if this will be enough to override the upstream config or if we need to redefine the package rules.